lib/components/viewers/realtime-status-label.js

```diff
@@ -34,11 +34,11 @@ const STATUS = {
     label: 'early'
   },
   LATE: {
-    color: '#d9534f',
+    color: '#D92923',
     label: 'late'
   },
   ON_TIME: {
-    color: '#5cb85c',
+    color: '#028602',
     label: 'on time'
   },
   SCHEDULED: {
```

Comment:
What color are we using for scheduled departures? We may need to check that we are using the same as in https://github.com/ibi-group/otp-ui/pull/8/files for consistency.

Reply:
From what I can gather, it's inherited from otp-ui? Or am I misunderstanding what's going on here

Reply:
The color is `#bbb` and defined underneath and gets merged in the CSS by the `Container` component. Not a critical item, and I don't think the Lighthouse devtools plugin can catch that as is as these labels are not part of the initial render of stop viewers.

Reply:
If `withBackground` is false, isn't it inherited? Otherwise yes I agree. May have to do this on a project by project basis. For what it's worth, Firefox dev tools can do color contrast checking even on subsequent renders!